src/queue.c

```diff
@@ -7818,6 +7818,7 @@ _dispatch_sigsuspend(void)
 }
 #endif // !defined(_WIN32)
 
+#if defined(__linux__)
 DISPATCH_NORETURN
```

Comment:
This thread exists on darwin too.

Reply:
The previous mistake resulted in this - a `#if 0` made me think it was unreferenced.